docs/android/remote-node.mdx

```diff
@@ -3,24 +3,64 @@ name: Remote Node Setup
 menu: Android
 ---
 
-# Remote Node Setup for Zap Android
+# Remote Node Setup
 
-You can use Zap Android to connect and control a remote LND node.
+You can use Zap Android to connect and control your own LND node. <br />
 
-### Step 1: Setting up a remote node
+In case you don't already have an own LND node, you can create one with the project found below.<br />
```

Comment:
> you can create one with the project found below

Probably should say "...with one of the projects..." since there are multiple listed